packages/models/src/eservice/eservice.ts

```diff
@@ -104,6 +128,13 @@ export const EServiceMode = z.enum([
 ]);
 export type EServiceMode = z.infer<typeof EServiceMode>;
 
+export const EServiceTemplateRef = z.object({
+  id: EServiceTemplateId,
+  instanceId: z.string().optional(),
```

Comment:
I have a general comment for this `instanceId`:
Reading the model at the beginning of the review, I expected it to be an ID: a UUID like all other ID fields in our codebase - although I did not fully understand what it was meant to identify.
However, if I understood correctly, it's actually the **name** of the instance, concatenated to the eservice name when creating/updating it.
Changing this would be big, so I don't propose doing it now - but could we track, if you agree, a task to improve this name? Something like `instanceName`

Reply:
Thank you very much Eric, I opened a discussion on the feature channel and we decided to update the name as you suggested.
[Here](https://github.com/pagopa/interop-be-monorepo/pull/1616)'s the PR